tests_integration/test_dev_server_get_patients.py

```diff
@@ -42,3 +42,17 @@ async def test_dev_server_get_patients() -> None:
     assert len(responses_) == 1
     assert responses_[0]["id"] == resource["id"]
     assert responses_[0]["resourceType"] == resource["resourceType"]
+
+    # test page_number can se set independent of page_size param
+    fhir_client._page_size = None
```

Comment:
these test files are specific to connecting to some sort of env endpoint like staging/dev. I think this test code should be moved to helix.fhir.client.sdk/tests